docs/index.rst

```diff
@@ -1,231 +1,8 @@
-table, which records monthly per-plant fuel deliveries, you may want to have the name
-of the plant alongside the fuel delivery information since it's more recognizable than
-the plant ID.
-
-Rather than requiring everyone to write their own SQL ``SELECT`` and ``JOIN`` statements
-or do a bunch of :func:`pandas.merge` operations to bring together data, PUDL provides a
-variety of output tables that contain all of the useful information in one place. In
-some cases, like with EIA, the output tables are composed to closely resemble the raw
-spreadsheet tables you're familiar with.
-
-The Output layer also contains tables produced by analytical routines for
-calculating derived values like the heat rate by generation unit (:meth:`hr_by_unit
-<pudl.output.pudltabl.PudlTabl.hr_by_unit>`) or the capacity factor by generator
-(:meth:`capacity_factor <pudl.output.pudltabl.PudlTabl.capacity_factor>`). We intend to
-integrate more analytical outputs into the library over time.
-
-.. seealso::
-
-    * `The PUDL Examples GitHub repo <https://github.com/catalyst-cooperative/pudl-examples>`__
-      to see how to access the PUDL Database directly, use the output functions, or
-      work with the EPA CEMS data using Dask.
-    * `How to Learn Dask in 2021 <https://coiled.io/blog/how-to-learn-dask-in-2021/>`__
-      is a great collection of self-guided resources if you are already familiar with
-      Python, Pandas, and NumPy.
-
-.. _test-and-validate:
-
----------------------------------------------------------------------------------------
-Data Validation
----------------------------------------------------------------------------------------
-We have a growing collection of data validation test cases that we run before
-publishing a data release to try and avoid publishing data with known issues. Most of
-these validations are described in the :mod:`pudl.validate` module. They check things
-like:
-
-* The heat content of various fuel types is within expected bounds.
-* Coal ash, moisture, mercury, sulfur, etc. content are within expected bounds
-* Generator heat rates and capacity factors are realistic for the type of prime mover
-  being reported.
-
-Some data validations are currently only specified within our test suite, including:
-
-* The expected number of records within each table
-* The fact that there are no entirely N/A columns
-
-A variety of database integrity checks are also run either during the data processing
-or when the data is loaded into SQLite.
-
-See our :doc:`dev/testing` documentation for more information.
-
----------------------------------------------------------------------------------------
-Organizations using PUDL
----------------------------------------------------------------------------------------
-.. include:: ../README.rst
-   :start-after: .. _pudl_orgs_start:
-   :end-before: .. _pudl_orgs_end:
-
-
-
 .. toctree::
   :hidden:
   :maxdepth: 2
 
+  Introduction <Introduction>
```

Comment:
```suggestion
  About PUDL <self>
```

This will create a reference to the page in the table of contents without needing to create a new page or having an empty index.